AnkiDroid/src/test/java/com/ichi2/anki/pages/PostRequestHandlerTest.kt

```diff
@@ -0,0 +1,47 @@
+/*
+ *  Copyright (c) 2024 voczi <dev@voczi.com>
+ *
+ *  This program is free software; you can redistribute it and/or modify it under
+ *  the terms of the GNU General Public License as published by the Free Software
+ *  Foundation; either version 3 of the License, or (at your option) any later
+ *  version.
+ *
+ *  This program is distributed in the hope that it will be useful, but WITHOUT ANY
+ *  WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+ *  PARTICULAR PURPOSE. See the GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License along with
+ *  this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package com.ichi2.anki.pages
+
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import com.ichi2.anki.RobolectricTest
+import com.ichi2.testutils.HamcrestUtils.containsInAnyOrder
+import org.hamcrest.MatcherAssert.assertThat
+import org.junit.Test
+import org.junit.runner.RunWith
+import java.io.InputStreamReader
+
+@RunWith(AndroidJUnit4::class)
+class PostRequestHandlerTest : RobolectricTest() {
+    @Test
+    fun `All backend typescript functions should be handled`() {
+        val assetManager = targetContext.assets
+        // read auto-generated list of all typescript funcs created+packaged during backend build
+        val tsFuncsStream = InputStreamReader(assetManager.open("backend/ts_funcs.txt"))
+        val tsFuncs = tsFuncsStream.readLines()
+        assertThat("List of stored Typescript functions is not empty", tsFuncs.isNotEmpty())
```

Comment:
Typically in Hamcrest, you typically don't want to pass in a boolean and instead want to pass in a matcher

```suggestion
        assertThat("Stored TypeScript functions", tsFuncs, not(empty()))
```

Which produces a better error:
```
Stored Typescript functions
Expected: not an empty collection
     but: was <[]>
java.lang.AssertionError: Stored Typescript functions
Expected: not an empty collection
     but: was <[]>
```

Note that `empty()` and `not()` are imports from `Hamcrest`